app/config.js

```diff
@@ -84,3 +88,31 @@ exports.getPlugins = function () {
     localPlugins: cfg.localPlugins
   };
 };
+
+exports.window = {
+  position: undefined,
+  size: undefined,
+  cwd: undefined,
+  get () {
+    let position = winCfg.get('windowPosition');
+    let size = winCfg.get('windowSize');
+    return {
+      position: (position !== undefined) ? position : [50, 50],
+      size: (size !== undefined) ? size : [540, 340]
+    };
+  },
+  set (position, size, cwd) {
+    this.position = position;
+    this.size = size;
+    this.cwd = cwd;
```

Comment:
Why save the `cwd`? From what I can see, you're actually not setting it anywhere.